packages/jest-types/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@types/sigstore-jest-extended",
+  "private": "true",
```

Comment:
This is the magic . . . putting this package in the `@types` namespace grants it special treatment in the TS world and marking it "private" means that it will never by published to npm.